### PR TITLE
updated changelog for JENKINS-32328

### PR DESCRIPTION
**Pull request by varmenise**

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -72,6 +72,9 @@
   <li class="rfe">
     Added missing translations for Polish language.
     (<a href="https://github.com/jenkinsci/jenkins/pull/1986">pull 1986</a>)
+  <li class="bug">
+    A custom update center is not consulted for Tool installer metadata.
```
Comment (oleg-nenashev) [inline changelog.html]: Would make sense to convert the message to "Retrieve tool installer metadata from all update sites" or whatever

Comment (mslusarczyk) [inline changelog.html]: +1

**Comment by varmenise** [comment]
@oleg-nenashev

**Comment by oleg-nenashev** [comment]
:bee: and :+1: 
@reviewbybees done

**Comment by daniel-beck** [comment]
@varmenise @oleg-nenashev The change was only in 1.647, while this PR attributes it to 1.646.

**Pull request merged**